src/_scss/pages/account/header/_options.scss

```diff
@@ -2,14 +2,18 @@
     @import "../../../mixins/moreOptionsBtn";
     @include display(flex);
     @include justify-content(space-between);
-    padding-top: $global-pad-mrg;
-    margin-top: $global-pad-mrg;
-    border-top: 1px solid rgba($color-white, .1);
+
+    @include flex(1 1 auto);
+
     ul.summary-status-items {
         @include unstyled-list;
+        padding: rem(5) 0;
+
         li {
-            display: inline-block;
+            @include display(flex);
```

Comment:
Since we're using flex now, can we get rid of `float: left;` and `text-align: right;` in the child elements? Same goes for tablet view.